### PR TITLE
IT-4682 user resource API

### PR DESCRIPTION
**Pull request by dev-alihaider-sc**
https://sellerchamp.atlassian.net/browse/IT-4682

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -1383,3 +1383,90 @@ curl --request POST \
...
+
+Make sure to replace `YOUR-TOKEN` with your API key token.
+```
+> The above command returns JSON structured like this:
+
+```json
+{
+  "user": {
+    "id": "61450d5c7305cf1e6583ef50",
+    "api_token": "c0bbf17e87440d1e1fdc497c9ea3b9ec",
```
Comment (noorani786) [inline source/index.html.md]: This is fake data or real data that is being exposed here?

Comment (noorani786) [reply source/index.html.md]: Also, dont send api token in the response. That's like sending password in cleartext.

File: source/index.html.md [hunk 2]
```diff
@@ -1383,3 +1383,90 @@ curl --request POST \
...
+------------- | ------------- | ---------------------------------
+push_notifications | Hash | Hash of values for notification settings containing allowed marketplace ids
+
+> The above command returns JSON structured like this:
+
+```json
+{
+  "user": {
+    "id": "61450d5c7305cf1e6583ef50",
+    "api_token": "c0bbf17e87440d1e1fdc497c9ea3b9ec",
```
Comment (noorani786) [inline source/index.html.md]: remove api-token .  make sure id value is fake.

File: source/index.html.md [hunk 3]
```diff
@@ -1383,3 +1383,90 @@ curl --request POST \
...
+    "name": "Nizar Noorani",
+    "client_name": "SC-Client",
+    "user_type": "client",
+    "push_notifications": {
+      "order_create": [
+        "MARKETPLACE_ACCOUNT_IDS"
```
Comment (noorani786) [inline source/index.html.md]: put some values here instead of 'MARKETPLACE_ACCOUNT_IDS'

**Review by noorani786 — CHANGES_REQUESTED** [review]

File: source/index.html.md (hunk 1 above)
Comment (dev-alihaider-sc) [reply source/index.html.md]: Data is from my local.
however, I updated it to dummy ones now.

**Pull request merged**